infrastructure/lambda/template.yaml

```diff
@@ -162,14 +161,6 @@ Mappings:
     production:
       dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceProductionVariables
 
-  MaximumAttemptCountMapping:
```

Comment:
Is this not being used anymore?

Reply:
No the parameter was removed in favour a constant in the handler.

https://github.com/govuk-one-login/ipv-cri-uk-passport-api/blob/ffad73c8f6c86508fe7e664861f73ecece37da92/lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandler.java#L64